src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks.js

```diff
@@ -0,0 +1,42 @@
+import { clone, pathOr } from 'ramda';
+import convertParagraph from './convertParagraph';
+
+const convertToOptimoBlocks = async jsonRaw => {
+  const handleMissingType = block =>
+    console.log(`Missing type field on block ${block.type}`); // eslint-disable-line no-console
+
+  const typesToConvert = {
+    paragraph: convertParagraph,
+  };
+
+  const parseBlockByType = async block => {
```

Comment:
Can you define these functions outside of `convertToOptimoBlocks` please? nested functions arnt really simorgh style